patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md

```diff
@@ -79,23 +92,31 @@ Also for decision making outside of pure software design decisions, transparent
 
 - **Uber** - According to this blog post by Gergely Orosz: [Scaling Engineering Teams via Writing Things Down and Sharing - aka RFCs][uber].
 - **Europace** - As described in Open Organization: [Setting cross-team standards and best practices in the open][europace].
+- **BBC iPlayer & Sounds** - As presented at the ISC Fall Summit 2020 [Using Internal RFCs to Enhance Collaboration][bbc].
+- **Google Design Docs** - As described in this blog post by Malte Ubl [Design Docs at Google][google]
```

Comment:
```suggestion
- **BBC iPlayer & Sounds** - As presented at the ISC Fall Summit 2020 [Using Internal RFCs to Enhance Collaboration][bbc].
- **Europace** - As described in Open Organization: [Setting cross-team standards and best practices in the open][europace].
- **Uber** - According to this blog post by Gergely Orosz: [Scaling Engineering Teams via Writing Things Down and Sharing - aka RFCs][uber].
- **Google Design Docs** - As described in this blog post by Malte Ubl [Design Docs at Google][google]
```

I vote for calling out the BBC first, as you are the one putting in the work to contribute this pattern.

Reply:
Sure, happy to take this change :)